integrationTests/chainSimulator/bridge/esdtSafe_test.go

```diff
@@ -136,7 +138,7 @@ func TestChainSimulator_ExecuteAndDepositTokensWithPrefix(t *testing.T) {
 		// register sovereign token identifier
 		// this will issue a new token on main chain and create a mapper between the identifiers
 		registerTokens(t, cs, wallet, &nonce, bridgeData.ESDTSafeAddress, token)
-		tokensMapper[token.Identifier] = chainSim.GetIssuedEsdtIdentifier(t, cs, getTokenTicker(token.Identifier), token.Type.String())
+		tokensMapper[token.Identifier] = chainSim.GetIssuedEsdtIdentifier(t, cs, getTokenTicker(token.Identifier), "")
```

Comment:
Why was this change needed?

Reply:
Yes, because the `.String()` is not working properly because of the enum